pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -572,6 +572,10 @@ type PDSpec struct {
 	// +kubebuilder:default=30
 	StartTimeout int `json:"startTimeout,omitempty"`
 
+	// Wait time before pd get started
+	// +kubebuilder:default=0
+	InitWaitTime int `json:"initWaitTime,omitempty"`
```

Comment:
How about adding a comment to explain this works with `WaitForDnsNameIpMatch`?

Reply:
@csuzhangxc added